server/configuration_test.go

```diff
@@ -104,11 +104,16 @@ func TestOnConfigurationChange(t *testing.T) {
 			api := test.SetupAPI()
 			defer api.AssertExpectations(t)
 
+			helpers := &plugintest.Helpers{}
```

Comment:
Work testing a failure to ensure the bot?

Reply:
Yes. I should do that.